rust/protocol/src/storage/inmem.rs

```diff
@@ -1,34 +1,41 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2021 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
+//! Implementations for stores defined in [super::traits].
```

Comment:
I think it's worth mentioning that they're "in-memory" or "ephemeral" or "for testing".

Reply:
Done!